src/adapter/z-stack/adapter/zStackAdapter.ts

```diff
@@ -6,6 +6,7 @@ import {Queue, Waitress, Wait} from '../../../utils';
 import {logger} from '../../../utils/logger';
 import {BroadcastAddress} from '../../../zspec/enums';
 import * as Zcl from '../../../zspec/zcl';
+import {Status} from '../../../zspec/zdo';
```

Comment:
Might want to import Zdo namespace to use `Zdo.Status`
`import * as Zdo from '../../../zspec/zdo';`
or rename
`import {Status as ZdoStatus} from '../../../zspec/zdo';`
to avoid confusion in the code.

(can probably replace several hardcoded values with consts/enums from `Zdo`, so the namespace will probably end up being better)

Reply:
Fixed!